client/state/happychat/selectors/test/get-skills.js

```diff
@@ -0,0 +1,115 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_GROUP_WPCOM,
+	HAPPYCHAT_GROUP_JPOP,
+	HAPPYCHAT_SKILLS,
+} from 'state/happychat/constants';
+import getSkills from 'state/happychat/selectors/get-skills';
+
+describe( '#getSkills()', () => {
+	let _window; // Keep a copy of the original window if any
```

Comment:
Why do we need to interact with the window object in these tests on selectors? I'm guessing this is copy-pasted in from other tests and was just never removed, but just making sure — we should delete it unless it's needed for some reason.

Reply:
Your guess was correct, this was copy pasted. I removed the window references as they are not used.